### PR TITLE
OBPIH-5026 Change naming in the product association import

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: grails-app/domain/org/pih/warehouse/product/ProductAssociation.groovy [hunk 1]
```diff
@@ -26,7 +26,7 @@ class ProductAssociation {
     String id
     ProductAssociationTypeCode code
     Product associatedProduct
-    BigDecimal quantity = 0
+    BigDecimal quantity = 1
```
Comment (jmiranda) [inline grails-app/domain/org/pih/warehouse/product/ProductAssociation.groovy:29]: This feels like a breaking change. What's the reason for this?

**Review by jmiranda — COMMENTED** [review]
Everything else looks good, but please respond to the comment I made.

File: grails-app/domain/org/pih/warehouse/product/ProductAssociation.groovy (hunk 1 above)
Comment (awalkowiak) [reply grails-app/domain/org/pih/warehouse/product/ProductAssociation.groovy:29]: To get rid of setting conversion by default to `1:0` (or `0:1`) which makes no sense

Comment (awalkowiak) [reply grails-app/domain/org/pih/warehouse/product/ProductAssociation.groovy]: I restored the 0 here, if it might be breaking

**Pull request merged**